From 56ff33e848d1a9b77f81e0d3b5ee0b79edcd4938 Mon Sep 17 00:00:00 2001
From: Minghuan Lian <Minghuan.Lian@nxp.com>
Date: Thu, 16 Feb 2017 17:52:08 +0800
Subject: [PATCH 06/13] arm64: dts: ls1043a: fix typo of MSI compatible string

Cherry-pick patchwork patch with context adjustment.

"1" should be replaced by "l". This is a typo.
The patch is to fix it.

Signed-off-by: Minghuan Lian <Minghuan.Lian@nxp.com>
Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
@@ -991,7 +991,7 @@
 		};
 
 		msi1: msi-controller1@1571000 {
-			compatible = "fsl,1s1043a-msi";
+			compatible = "fsl,ls1043a-msi";
 			reg = <0x0 0x1571000 0x0 0x4>,
 			      <0x0 0x1571004 0x0 0x4>;
 			reg-names = "msiir", "msir";
@@ -1000,7 +1000,7 @@
 		};
 
 		msi2: msi-controller2@1572000 {
-			compatible = "fsl,1s1043a-msi";
+			compatible = "fsl,ls1043a-msi";
 			reg = <0x0 0x1572000 0x0 0x4>,
 			      <0x0 0x1572004 0x0 0x4>;
 			reg-names = "msiir", "msir";
@@ -1009,7 +1009,7 @@
 		};
 
 		msi3: msi-controller3@1573000 {
-			compatible = "fsl,1s1043a-msi";
+			compatible = "fsl,ls1043a-msi";
 			reg = <0x0 0x1573000 0x0 0x4>,
 			      <0x0 0x1573004 0x0 0x4>;
 			reg-names = "msiir", "msir";
